### PR TITLE
Template text: re-phrase various screen message in preparation of i18…

### PR DESCRIPTION
**Pull request by sunweaver**
…n work.

### CODE REVIEW

File: personal/privacyidea/default/mfa_editToken.tpl [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *}
-<h2>{t escape=no 1=$mfa_{$token.tokentype}_icon 2=$mfa_{$token.tokentype}_title}<i class="material-icons">%1</i> %2 details{/t}</h2>
+<h2><i class="material-icons">{t escape=no 1=$mfa_{$token.tokentype}_icon}%1{/t}</i> {t 2=$mfa_{$token.tokentype}_title}%2 - Details{/t}</h2>
```
Comment (dzatoah) [inline personal/privacyidea/default/mfa_editToken.tpl:20]: Why translate icon names in the first place?

File: personal/privacyidea/default/mfatype_totp/phase00.tpl [hunk 2]
```diff
@@ -25,7 +25,7 @@
     <p>{t}An authentication app on your smartphone generates time-based one-time passwords (TOTP) which must be entered on each login.{/t}</p>
     <p>{t}Setting up the authentication app requires multiple steps:{/t}</p>
     <ol>
-        <li>{t}Install and open a suitable TOTP authentication app.{/t}</li>
+        <li>{t}Install and open a suitable TOTP authentication app{/t}.</li>
```
Comment (dzatoah) [inline personal/privacyidea/default/mfatype_totp/phase00.tpl:28]: Why this change?

File: personal/privacyidea/default/mfa_editToken.tpl (hunk 1 above)
Comment (dzatoah) [reply personal/privacyidea/default/mfa_editToken.tpl:20]: This breaks mfa_editToken.tpl! Please revert this change.

File: personal/privacyidea/default/mfatype_totp/phase00.tpl (hunk 2 above)
Comment (sunweaver) [reply personal/privacyidea/default/mfatype_totp/phase00.tpl:28]: This makes translating the customer theme easier.
